project.clj

```diff
@@ -7,19 +7,19 @@
                  [marick/clojure-commons "3.0.0" :exclusions [org.clojure/clojure]]
                  ;; structural-typing currently broken with specter 0.13
                  ;; [marick/structural-typing "2.0.4" :exclusions [org.clojure/clojure org.clojure/clojurescript]]
-                 [org.clojure/math.combinatorics "0.1.3"]
+                 [org.clojure/math.combinatorics "0.1.4"]
```

Comment:
Note that this version bump drops support for for Clojure 1.2 - 1.6. 
Does midje support those versions?